rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/data_definitions.kt

```diff
@@ -196,9 +198,9 @@ internal fun RpgParser.Parm_fixedContext.toAst(
                     StringType(elementSize!!, varying)
                 }
             }
-            "A" -> StringType(elementSize!!, varying)
-            "N" -> BooleanType
-            "Z" -> TimeStampType
+            RpgType.CHARACTER.rpgType -> StringType(elementSize!!, varying)
```

Comment:
Does "A", "N", "Z" refer to the definition of variables in the rpgle program?
Why did you replace "A" with "RpgType.CHARACTER.rpgType" and so on?

Reply:
Because those strings were already defined as RpgType and in in a switch is better to work with the enum rather than string literal.